sourcers/json/json.go

```diff
@@ -0,0 +1,62 @@
+package json
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"strings"
+
+	"github.com/ardanlabs/conf"
+)
+
+type JSONSourcer struct {
```

Comment:
Missing comment. Please make sure this is clean with staticcheck.

Reply:
This is clean according to both `staticcheck` and `golangci-lint`.  Should there be a lint error due to no comment?